src/llama-vocab.cpp

```diff
@@ -321,6 +321,21 @@ struct llm_tokenizer_spm {
 
 // TODO: there are a lot of common parts between spm and bpe tokenizers, should be refactored and reused
 
+template<typename T, typename Container = std::vector<T>, typename Compare = std::less<typename Container::value_type>>
+class MovablePriorityQueue : public std::priority_queue<T, Container, Compare> {
```

Comment:
Change the name to `snake_case`:

```suggestion
class llama_priority_queue : public std::priority_queue<T, Container, Compare> {
```